src/main/java/com/example/auctionapp/enumeration/ColorEnum.java

```diff
@@ -0,0 +1,15 @@
+package com.example.auctionapp.enumeration;
+
+public enum ColorEnum {
+    Black,
+    White,
+    Grey,
+    Red,
+    Yellow,
+    Blue,
+    Purple,
+    Green,
+    Pink,
+    Orange,
+    Brown
```

Comment:
This would be the use case for some business logic that requires enums with different notations.
The way we can implement this ENUM is to:

- have the server only work with UPPER_CASE enums
- have the client contain this type of enum representation (UPPER_CASE("CamelCase")), on the UI we can show Camel Case, but send the UPPER_CASE notation to the api

Or

- have the API accept the client UI values, and  use `.fromValue("sent value")` to construct enums.